config/initializers/devise.rb

```diff
@@ -0,0 +1,265 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '4b0981771fdb1a6fccb92c3f17cdf946dc0bb521b3afd007b2f785d07ac82aac638811ce7ce71cf021ad5652df3a07529a702ad7e690183e3b07bfca131b7d1e'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Eventually, when the client buys a domain name, you'll need to change that to something like `contact@swimclub.net`. Worth adding that task to a Trello card.